README.md

```diff
@@ -226,9 +226,9 @@ likely get blanks or � (U+FFFD, REPLACEMENT CHARACTER) for missing characters,
 and subsequent characters on the line may be misplaced.
 
 It is worth knowing that several terminals draw the block characters directly,
-rather than loading them from a font. This is generally desirable. Quadrants
-and sextants are not the place to demonstrate your design virtuosity. To
-inspect your environment's rendering of drawing characters, run
+rather than loading them from a font. This is generally desirable. Quadrants,
+sextants, and octants are not the place to demonstrate your design virtuosity.
```

Comment:
i'm guessing your methodology for this kind of thing involved grepping for at a minimum `sextant`. i'm trying to think of other stuff we'd want to check. `NCBLIT` is broad but probably a good one. i'll run these checks, just thinking aloud.

Reply:
Correct. I looked for `sextant`, but not for `NCBLIT`.